src/style-spec/reference/v8.json

```diff
@@ -227,6 +225,58 @@
       "doc": "Other keys to configure the data source."
     }
   },
+  "source_raster_dem": {
+    "type": {
+      "required": true,
+      "type": "enum",
+      "values": {
+        "raster-dem": {
+            "doc": "A raster DEM source using Mapbox Terrain RGB"
+        }
+      },
+      "doc": "The type of the source."
+    },
+    "url": {
+      "type": "string",
+      "doc": "A URL to a TileJSON resource. Supported protocols are `http:`, `https:`, and `mapbox://<mapid>`."
+    },
+    "tiles": {
+      "type": "array",
+      "value": "string",
+      "doc": "An array of one or more tile source URLs, as in the TileJSON spec."
+    },
+    "bounds": {
```

Comment:
That's interesting, I tested bounds and found it had no affect. I better re-test it.

Reply:
in order for it to work you have to use the `tiles: [...]` property to specify the url because otherwise the `bounds` returned by the source's `TileJSON` will override the ones you specify. which I suppose we should also document 😅

Reply:
this worked as expected for me:
```es6
        "rasterTerrain": {
            "type": "raster-dem",
            "tiles": [`https://a.tiles.mapbox.com/v4/mapbox.terrain-rgb/{z}/{x}/{y}.png?access_token=${mapboxgl.accessToken}`],
            "bounds" : [-40, -40, 40, 40]
        }
```

Reply:
ahh that makes sense now :grin:

Reply:
Agreed, it should be documented but I don't think it's critical for this PR. The same applies to the rest of these properties and raster sources too.